src/main/java/seedu/address/logic/commands/DeletePropertyCommand.java

```diff
@@ -46,7 +48,16 @@ public DeletePropertyCommand(PostalCode postalCode, Unit unitNumber) {
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        return new CommandResult(MESSAGE_DELETE_PROPERTY_SUCCESS);
+
+        Property propertyToDelete = model.getFilteredPropertyList().stream()
```

Comment:
caveat that this looks like a violation of the Law of Demeter. Maybe we should add a method `toDelete(postalCode, unitNumber)` instead. I think DeleteBuyerCommand was implemented similarly though. We can leave it here first for v1.3 I think